### PR TITLE
Engdocs 1472

### PR DESCRIPTION
**Pull request by aevesdocker**
PR for DD 4.21 content:
- VirtioFS is default filesharing for new installs
- automatic pause for DD
- CLI hints and tips in DD settings 
- Global Search added functionality - docs tab

### CODE REVIEW

File: desktop/settings/linux.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ If you choose the integrated terminal, you can run commands in a running contain
 - **Use Docker Compose V2**. Select to enable the `docker-compose` command to
   use Docker Compose V2. For more information, see [Migrate to Compose V2](../../compose/migrate.md).
 
+- **Show CLI hints**. CLI hints and tips when running Docker commands in the CLI.
```
Comment (crazy-max) [inline desktop/settings/linux.md:45]: Would appreciate if we can stipulate how to disable hints related to https://github.com/docker/compose-cli/pull/2252?

TLDR; "To disable CLI hints, you can set `DOCKER_CLI_HINTS` env var to `false`".

Comment (crazy-max) [reply desktop/settings/linux.md:45]: Even better if we can add this to the DD 4.21 release notes.

Comment (aevesdocker) [reply desktop/settings/linux.md:45]: Thanks @crazy-max, will get that added in. And I see you already beat me to it on the release notes 👍

File: desktop/use-desktop/pause.md [hunk 2]
```diff
@@ -4,14 +4,10 @@ keywords: Docker Dashboard, manage, containers, gui, dashboard, pause, user manu
...
+To save CPU resources on your machine, Docker Desktop automatically pauses when it is not in use. 
 
-From the Docker menu, select![whale menu](../images/whale-x.svg){: .inline} and then **Pause** to pause Docker Desktop.
+When  Docker Desktop is paused, the Linux VM running Docker Engine is paused, the current state of all your containers are saved in memory, and all processes are frozen. This reduces the CPU usage and helps you retain a longer battery life on your laptop. 
 
-Docker Desktop displays the paused status on the Docker menu and on the  **Containers**, **Images**, **Volumes**, and **Dev Environment** screens in Docker Dashboard. You can still access the **Settings** and the **Troubleshoot** menu from the Dashboard when you've paused Docker Desktop.
+You can also manually pause Docker Desktop. From either the menu bar on Mac or the tray icon on Windows, select the Docker menu ![whale menu](../images/whale-x.svg){: .inline} and then **Pause**. You can manually resume Docker Desktop by clicking the **Resume** option in the Docker menu, or by running any commands in the Docker CLI.
```
Comment (dvdksn) [inline desktop/use-desktop/pause.md:11]:
```suggestion
You can also manually pause Docker Desktop. From either the menu bar on Mac or the tray icon on Windows, select the Docker menu ![whale menu](../images/whale-x.svg){: .inline} and then **Pause**. You can manually resume Docker Desktop by clicking the **Resume** option in the Docker menu, or by running any Docker CLI command.
```

?

File: desktop/use-desktop/pause.md [hunk 3]
```diff
@@ -4,14 +4,10 @@ keywords: Docker Dashboard, manage, containers, gui, dashboard, pause, user manu
 title: Pause Docker Desktop
 ---
 
-You can pause your Docker Desktop session when you are not actively using it and save CPU resources on your machine. When you pause Docker Desktop, the Linux VM running Docker Engine is paused, the current state of all your containers are saved in memory, and all processes are frozen. This reduces the CPU usage and helps you retain a longer battery life on your laptop. You can resume Docker Desktop when you want by clicking the **Resume** option.
+To save CPU resources on your machine, Docker Desktop automatically pauses when it is not in use. 
 
-From the Docker menu, select![whale menu](../images/whale-x.svg){: .inline} and then **Pause** to pause Docker Desktop.
+When  Docker Desktop is paused, the Linux VM running Docker Engine is paused, the current state of all your containers are saved in memory, and all processes are frozen. This reduces the CPU usage and helps you retain a longer battery life on your laptop.
```
Comment (dvdksn) [inline desktop/use-desktop/pause.md:9]:
```suggestion
When Docker Desktop is paused, the Linux VM running Docker Engine is paused, the current state of all your containers are saved in memory, and all processes are frozen. This reduces the CPU usage and helps you retain a longer battery life on your laptop. 
```

**Review by dvdksn — APPROVED** [review]
Nonblocking nits

File: desktop/use-desktop/pause.md (hunk 2 above)
Comment (aevesdocker) [reply desktop/use-desktop/pause.md:11]: Yes, much neater

**Review by dvdksn — APPROVED** [review]
